### ISSUE TITLE
I didn’t find icon choice intuitive

### ISSUE THREAD

**Issue by andylolz** [OpenDataServices/org-ids#147]
I didn’t expect the `map-zoom-out` (or ‘fullscreen’) icon to take me to the expanded view for the list (I don’t know what I thought it would do.)

![screen shot 2017-08-09 at 14 53 20](https://user-images.githubusercontent.com/464193/29125033-87b1be08-7d12-11e7-9ebb-181ca3b9f528.png)

I actually expected one of the icons to copy the prefix to the clipboard for me, but that is perhaps a bit of a silly feature request.

**Comment by BobHarper1** [comment]
Would a google style three dots ... be more intuitive? I don't particularly think it is, but I like the three dashes https://thenounproject.com/search/?q=menu&i=1211950

**Comment by andylolz** [comment]
> the three dashes https://thenounproject.com/search/?q=menu&i=1211950

A [hamburger](https://en.wikipedia.org/wiki/Hamburger_button), you mean? Not sure… I think it would be an improvement, but I suspect it might be weird here.

> a google style three dots

Like a [horizontal ellipsis](https://en.wikipedia.org/wiki/Ellipsis)? Yeah, I prefer this idea.

**Comment by BobHarper1** [comment]
That was my feelings about the ellipsis, it can be missed. Maybe an info circle? (To use its correct name ;)
 http://fontawesome.io/icon/info-circle/

**Comment by andylolz** [comment]
> That was my feelings about the ellipsis, it can be missed. Maybe an info circle?

Yeah – of the three, that’s my favourite! (and all of them are better than the fullscreen symbol imo.) Sorry not to offer any suggestions of my own!
